scripts/zones/Pashhow_Marshlands/mobs/BoWho_Warmonger.lua

```diff
@@ -3,16 +3,64 @@
 --   NM: Bo'Who Warmonger
 -----------------------------------
 require("scripts/globals/regimes")
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Pashhow_Marshlands/IDs")
 -----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
+
+entity.onMobSpawn = function(mob)
+    -- Takes half damage from all attacks
+    mob:addMod(xi.mod.DMG,-5000)
+
+    -- May spawn in a party with two other Quadav
+    if math.random(1,2) == 1 then
+        GetMobByID(ID.mob.BOWHO_WARMONGER + 1):setSpawn(mob:getXPos()+2, mob:getYPos(), mob:getZPos())
+        GetMobByID(ID.mob.BOWHO_WARMONGER + 2):setSpawn(mob:getXPos()+4, mob:getYPos(), mob:getZPos())
+        SpawnMob(ID.mob.BOWHO_WARMONGER + 1)
+        SpawnMob(ID.mob.BOWHO_WARMONGER + 2)
+    end
+end
+
+entity.onMobEngaged = function(mob, target)
+    local mobId = mob:getID()
+    for i = 1, 2 do
+        GetMobByID(mobId+i):updateEnmity(target)
+    end
+end
+
+entity.onMobRoam = function(mob)
+    local mobId = mob:getID()
+
+    for i = 1, 2 do
+        local guard = GetMobByID(mobId + i)
+        if guard:isSpawned() and guard:getID() == mobId + 1 then
+            guard:pathTo(mob:getXPos() + 1, mob:getYPos() + 3, mob:getZPos() + 0.15)
+        elseif guard:isSpawned() and guard:getID() == mobId + 2 then
+            guard:pathTo(mob:getXPos() + 3, mob:getYPos() + 5, mob:getZPos() + 0.15)
+        end
+    end
+end
+
+entity.onMagicCastingCheck = function(mob, target, spell)
+    -- Frequently casts Cure 3 on itself below 50% HP
+    if mob:getHPP() < 50 then
```

Comment:
A better way to do this might be to change its cast priority modifier for heals
https://github.com/LandSandBoat/server/blob/base/scripts/globals/status.lua#L2406

unsure if you need to adjust the other types or not, but that'd let the already existing core code for chance to cast to handle the randomness

Reply:
Adjusted this to be an 80% chance for heals below 80%, otherwise does 40% which I see if the default for NMs

Reply:
if this is working out for you the way I hoped, then you prolly want to do the same to Demonic Tiphia